### DISCUSSION TITLE
All workspaces get moved to primary screen after wake from sleep

### DISCUSSION THREAD

**Discussion by rolling-thundar** [regolith-linux/regolith-desktop#931] [Q&A] [answered]
Hi

After I wake my machine from sleep, all the workspaces I had on my secondary screen get moved to my primary.

How can I change this behaviour in regolith, so that the workspaces remain where they were?

I'm on Regolith 3.0 (X11), on PopOS.

**Comment by kgilmer** [comment, marked as answer]
Hi @rolling-thundar , I have also had this behavior in the past.  I believe it to be related to graphics or hardware device hotplug.  I do not have a proper solution, but when dealing with this problem used a program to save/restore my layout.  This script or something similar may work for you as a work around: https://github.com/JonnyHaystack/i3-resurrect

**Comment by rolling-thundar** [comment]
Hey @kgilmer. Thank you for getting back to me. I will try i3-resurrect.

How did you integrate it's configuration into regolith?
